asmcli/asm_vm

```diff
@@ -945,7 +945,7 @@ uses_environ_workload_identity() {
   fi
 
   if [[ "$(retry 2 kubectl get configmap "${CONFIGMAP}" -n istio-system \
-    -ojsonpath='{.data.mesh}' | grep -x -c "^trustDomain: ${PROJECT_ID}.hub.id.goog$" \
+    -ojsonpath='{.data.mesh}' | grep -x -c "^trustDomain: ${PROJECT_ID}.\\(hub\\|svc\\).id.goog$" \
```

Comment:
Maybe. However, -E is one of those flags that has a slightly different meaning between the GNU and BSD grep implementations. I never bothered to learn what all of the differences are, so I generally just avoid it to make sure that it's compatible. Otherwise it won't work on most Macs.

Reply:
Makes sense!